examples/events.py

```diff
@@ -0,0 +1,123 @@
+"""
+Example showing events and clicking on meshes.
+
+Clicking on a cube will select it. Hovering the mouse over a cube
+will show a hover color. Double-clicking a cube will select all
+the items from that group (because the group has a double-click
+event handler).
+"""
+
+from random import randint, random
+
+from wgpu.gui.auto import WgpuCanvas, run
+import pygfx as gfx
+
+
+canvas = WgpuCanvas()
+renderer = gfx.renderers.WgpuRenderer(canvas)
+
+camera = gfx.PerspectiveCamera(70, 16 / 9)
+camera.position.z = 400
+
+controls = gfx.OrbitControls(camera.position.clone())
+controls.add_default_event_handlers(renderer, canvas, camera)
+
+scene = gfx.Scene()
+
+geometry = gfx.box_geometry(40, 40, 40)
+material = gfx.MeshPhongMaterial()
```

Comment:
renaming this to `default_material` would make some parts of this example more easy to read and understand